themes/kiali/layouts/faq/list.html

```diff
@@ -16,7 +16,7 @@ <h2><a href="{{ .Permalink }}">{{ .LinkTitle }}</a></h2>
       <ul>
```

Comment:
Probably you also need to change line num. 15 to also fix the headings.

This fix was easier than I thought :)

Reply:
Oopsie! Good catch :)